.circleci/config.yml

```diff
@@ -2,31 +2,26 @@ version: 2
 jobs:
   api:
     docker:
-      - image: python:3.7
+      - image: circleci/python:3.8
     working_directory: ~/repo/api
     steps:
       - checkout:
           path: ~/repo
       - restore_cache:
-          key: api-deps-v1-{{ checksum "requirements.txt" }}
-      - run:
-          name: Activate virtualenv
-          command: |
-            python -m venv venv
-            . ./venv/bin/activate
+          key: api-deps-v1-{{ checksum "poetry.lock" }}
       - run:
           name: Install Deps
-          command: pip install -r requirements.txt
+          command: poetry install
       - save_cache:
-          key: api-deps-v1-{{ checksum "requirements.txt" }}
+          key: api-deps-v1-{{ checksum "poetry.lock" }}
           paths:
             - venv
```

Comment:
Is this still the right Path? Does poetry put the venv here?

Reply:
Yes

Reply:
This log from the build makes me think otherwise:
`Creating virtualenv ttbud-zxzmhbhl-py3.8 in /home/circleci/.cache/pypoetry/virtualenvs`